Server/src/loaders/loadStartupProcess.ts

```diff
@@ -45,16 +46,15 @@ class loadStartupProcess {
     /**
      * Routes are added/loaded to the application here. All routes can be added following the style of fileUploadRouter
      */
-    app.use('/', datasetController)
     app.use('/', fileUploadRouter)
 
 
     /**
      * The following starts the server on port 4000 
      */
-    const port: number = Number(process.env.PORT) || 4000;
+    const port: Number = Number(process.env.PORT) || 4000;
     const startServer = async () => {
-      app.listen(port, () => {
+      await app.listen(port, () => {
```

Comment:
I would interpolate the actual port variable in here, so that the string is not hardcoded. i.e. `runniing on localhost:{port}`

Reply:
Was planning on moving all env variables after sql migration as Im not sure which will be kept and which not